### DISCUSSION TITLE
Subscription Model Change: Perpetual License

### DISCUSSION THREAD

**Discussion by btrsco** [spatie/ray#639] [Ideas]
Seems like the norm I've seen in the development software is that **when you purchase a license for a year** you get a perpetual license fixed at the version I bought in at or the last update before my year of updates ended.

I know you guys probably went with this subscription model on purpose but I would love to see a perpetual license as I don't see the need to continuously renew this software yearly if it already does what I need and doesn't have any bugs I've noticed. Please consider changing your subscription model to include a perpetual license for yearly subscriptions as well as maybe a renewal discount. 

Beyond Code ([Invoker](https://invoker.dev/#plans) and [Tinkerwell](https://tinkerwell.app/) have perpetual licensing) and JetBrains ([All products have perpetual licensing](https://www.jetbrains.com/phpstorm/buy/#personal)) have similar subscription models.

**Comment by freekmurze** [comment]
We will currently not be making any changes on this. We sometimes temporarily offer lifetime licenses. Keep an eye out on our Twitter to be able to get one of these, should we run that promotion again.
